packages/lib/src/interactions/Pan.tsx

```diff
@@ -25,18 +24,13 @@ function Pan(props: Props) {
     disabled,
   } = props;
 
-  const modifierKeys = getModifierKeyArray(modifierKey);
-  const shouldInteract = useInteraction(id, {
-    button,
-    modifierKeys,
-    disabled,
-  });
+  const shouldInteract = useInteraction(id, { button, modifierKey, disabled });
 
   const camera = useThree((state) => state.camera);
   const moveCameraTo = useMoveCameraTo();
 
   const startOffsetPosition = useRef<Vector3>(); // `useRef` to avoid re-renders
-  const isModifierKeyPressed = useModifierKeyPressed(modifierKeys);
+  const isModifierKeyPressed = useModifierKeyPressed(modifierKey);
```

Comment:
The array casting is now done internally, to avoid having to call and export `getModifierKeyArray`.